pytest/lib/utils.py

```diff
@@ -121,3 +122,23 @@ def count(self, pattern):
 def load_binary_file(filepath):
     with open(filepath, "rb") as binaryfile:
         return bytearray(binaryfile.read())
+
+def compile_rust_contract(content):
+    test_contract_rs = os.path.join(os.path.dirname(__file__), '../empty-contract-rs')
+    p = run('bash', input=f'''
+mkdir -p /tmp/near
```

Comment:
We should probably use https://docs.python.org/3.7/library/tempfile.html to avoid potential interference if we run tests concurrently.

Reply:
good point